simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp

```diff
@@ -563,6 +565,27 @@ void SimplePlanningSimulator::publish_acceleration()
   pub_acc_->publish(msg);
 }
 
+void SimplePlanningSimulator::publish_imu()
+{
+  sensor_msgs::msg::Imu imu;
+  imu.header.frame_id = "/base_link";
```

Comment:
/base_link -> base_link ?

Reply:
addressed in https://github.com/autowarefoundation/autoware.universe/pull/2843/commits/8bb8b70a55e4edaa791683ed407043c8c9c11d38